src/parser/parsers/ability_ui.py

```diff
@@ -158,8 +161,12 @@ def _parse_main_block(self, info_section, parsed_ability):
             title = None
             title_key = props.get('m_strPropertiesTitleLocString')
             if title_key is not None:
-                # localization keys are prefixed with a "#"
-                title_key = title_key.replace('#', '')
+                # localization keys are prefixed with a "#", remove it
+                if title_key.startswith('#'):
+                    title_key = title_key[len('#'):]
+                else:
+                    # There are some cases of this
+                    title_key = title_key.replace('#', '')
```

Comment:
@Surxe I don't see how this is needed. Are there any cases where the `#` is not at the beginning?

Reply:
Its not needed currently, just guards against potential valve changes.

Reply:
Right, that's all good. But this implies that a future key would have a # in the middle of the word. I will just revert it to keep the code clearer

Reply:
Oh sorry I was referring to the first one I added where theres currently no exceptions. Your right for this one its a pointless addition